appyx-components/experimental/puzzle15/common/src/commonMain/kotlin/com/bumble/appyx/components/experimental/puzzle15/ui/Puzzle15MotionController.kt

```diff
@@ -27,16 +28,17 @@ class Puzzle15MotionController(
     defaultAnimationSpec = defaultAnimationSpec
 ) {
 
+    private val twoThirds = 2f / 3f
+
     override fun Puzzle15Model.State.toUiTargets(): List<MatchedTargetUiState<Tile, TargetUiState>> {
-        val width = uiContext.transitionBounds.widthDp.value / 4
         return items.mapIndexed { index, tileElements ->
             MatchedTargetUiState(
                 element = tileElements,
                 targetUiState = TargetUiState(
-                    position = Position.Target(
-                        offset = DpOffset(
-                            x = (index % 4 * width).dp,
-                            y = (index / 4 * width).dp
+                    position = PositionInside.Target(
+                        alignment = BiasAlignment.InsideAlignment(
+                            -1f + (index % 4) * twoThirds,
+                            -1f + (index / 4) * twoThirds
```

Comment:
Let's simplify this by moving the calculation inside the alignment class.
```suggestion
                    position = PositionInside.Target(
                        alignment = BiasAlignment.InsideAlignment(
                            InsideAlignment.percentileAlignment(index % 4),
                            InsideAlignment.percentileAlignment(index / 4)
```